### ISSUE TITLE
Failed in SearchClient.SearchAsync(8911)

### ISSUE THREAD

**Issue by Seng-Jik** [Xwilarg/NHentaiSharp#2]
This error occurred on November 20, 2021.

```fsharp
> SearchClient.SearchAsync(8911).Result;;
```

```
System.AggregateException: One or more errors occurred. (Invalid format '0')
 ---> System.ArgumentException: Invalid format '0'
   at NHentaiSharp.Search.Page..ctor(Object json, String urlBase, String urlPreview)
   at CallSite.Target(Closure , CallSite , Type , Object , String , String )
   at NHentaiSharp.Search.GalleryElement..ctor(Object json)
   at CallSite.Target(Closure , CallSite , Type , Object )
   at NHentaiSharp.Search.SearchResult..ctor(Object json)
   at NHentaiSharp.Core.SearchClient.SearchAsync(Int32 page)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at System.Threading.Tasks.Task`1.get_Result()
   at <StartupCode$FSI_0007>.$FSI_0007.main@()
```

**Comment by Xwilarg** [comment]
Thanks for the bug report
I tried to reproduce it but there is no issue on my side, does is still don't work for you?

**Comment by Seng-Jik** [comment]
Now is 8912:

```fsharp
> SearchClient.SearchAsync(8912).Result;;
System.AggregateException: One or more errors occurred. (Invalid format '0')
 ---> System.ArgumentException: Invalid format '0'
   at NHentaiSharp.Search.Page..ctor(Object json, String urlBase, String urlPreview)
   at CallSite.Target(Closure , CallSite , Type , Object , String , String )
   at NHentaiSharp.Search.GalleryElement..ctor(Object json)
   at CallSite.Target(Closure , CallSite , Type , Object )
   at NHentaiSharp.Search.SearchResult..ctor(Object json)
   at NHentaiSharp.Core.SearchClient.SearchAsync(Int32 page)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at System.Threading.Tasks.Task`1.get_Result()
   at <StartupCode$FSI_0006>.$FSI_0006.main@()
```

**Comment by Xwilarg** [comment]
Sorry for the late answer, this is now fixed with e58588be9d1197076a5a5d34d70db9d3054650d6, I'll upload a new version of the library later today

This is actually related to a page being broken in NHentai (so the API returns an invalid format)
![image](https://user-images.githubusercontent.com/20053454/143447931-440cc6b1-b48d-4ff1-ae28-8a04adc5b32d.png)
